scenes/cslib_string.txt

```diff
@@ -314,3 +314,61 @@
 		*goto consists_of_char_list_loop
 	*set cslib_ret false
 *return
+
+
+*comment EXTRACT_TOKENS
+*comment ------------------------------
+*comment Split a given string using a separator.
+*comment The tokens are stored in a given array, up to its max.
```

Comment:
Do you think we should refer to the count and max attributes as '_max' and '_count' respectively? I feel if we're consistent about that, it might help indicate what we mean...? Maybe.